plugins/window-rules/window-rules.cpp

```diff
@@ -26,13 +26,15 @@ class wayfire_window_rules_t : public wf::plugin_interface_t
     void apply(const std::string & signal, wf::signal_data_t *data);
 
   private:
+    void setup_rules_from_config();
     wf::lexer_t _lexer;
 
     wf::signal_callback_t _created;
     wf::signal_callback_t _maximized;
     wf::signal_callback_t _unmaximized;
     wf::signal_callback_t _minimized;
     wf::signal_callback_t _fullscreened;
+    wf::signal_callback_t reload_config;
```

Comment:
A small thing: `wf::signal_callback_t` is deprecated, new code should use `wf::signal_connection_t` (and also whenever possible, please initialize lambda as soon as you declare it, I think it should be possible here).

Reply:
Ah I see. I will make changes and make a commit when I'm a little free. I more or less copied the code from commands plugin.